src/app/containers/Image/index.jsx

```diff
@@ -76,12 +82,16 @@ const ImageContainer = ({ blocks }) => {
         src={rawImageSrc}
         width={width}
         srcset={srcSet}
-        lazyLoad
+        lazyLoad={lazyLoad}
       />
     </Wrapper>
   );
 };
 
 ImageContainer.propTypes = imageModelPropTypes;
 
+ImageContainer.defaultProps = {
+  position: [0],
```

Comment:
```suggestion
  position: [1],
```

This should probably be `1` as the position content start from 1 unlike the array indexing which starts from `0`